django/template/loader_tags.py

```diff
@@ -207,6 +210,11 @@ def render(self, context):
         except Exception:
             if context.template.engine.debug:
                 raise
+            else:
+                template_name = getattr(context, 'template_name', 'unknown')
+                logger.warn('Exception raised while rendering {{% include %}} for '
```

Comment:
Yes, we should probably add an actual message like you've done here and use `exc_info=True`. Please do it in a separate commit or pull request using "Refs #18773 -- ..." in the commit message (for reference, the commit is dc5b01ad05e50ccde688c73c2ed3334a956076b0). Thanks!